app/v1/lifecycle/systems/endpoints.py

```diff
@@ -0,0 +1,71 @@
+from datetime import date
+
+from fastapi import APIRouter
+
+v1_router = APIRouter()
+
+OS_DATA_MOCKED = [
```

Comment:
Put this in `data/[something].py` similar to [this](https://github.com/RedHatInsights/digital-roadmap-backend/pull/16/files#diff-28e5486514290d044ceb29f95bba42b2c4e2db5767b13777cff5522bc558f7c6). That helps to keep the data at least somewhat separate from the code.